### ISSUE TITLE
Whisper align bollywood

### ISSUE THREAD

**Issue by Isyankar70** [rakuri255/UltraSinger#180]
Hi, ultrasinger ist verry good 

I have a problem when i convert a bollywood song i have the hindi language i have test 
With whisper align but i dont create it

**Comment by rakuri255** [comment]
Hi, did you try an hindi model from Huggingface?

**Comment by Isyankar70** [comment]
Yes i have try two hindi model the problem ist the normal translation make in hindi language but i will in romanized language but ist not translation

**Comment by rakuri255** [comment]
https://github.com/rakuri255/UltraSinger/issues/181
